frameworks/kafka/src/main/java/com/mesosphere/sdk/specification/RawTask.java

```diff
@@ -0,0 +1,179 @@
+package com.mesosphere.sdk.specification.yaml;
```

Comment:
Is this file needed with `rc2`?

Reply:
in rc2 i could only find changes needed to be done is to remove marathon enforce role and not to use shm-size when ipc is shared_parent , i am not sure if there was any change to ServiceSpec.java or RawTask.java to get rid of dependency from Raw.Task.java

Reply:
but i feel once new sdk update goes to master , we might not need this

Reply:
This file supposedly is not needed at all based on [this comment](https://github.com/mesosphere/dcos-commons/pull/3134/files#r307111134)

Reply:
Actually the issue is this: https://github.com/mesosphere/dcos-kafka-service/pull/440#issuecomment-515798610

You need to set the version which includes the changes in this file. So you don't need to add it yourself.

Reply:
Actually this file _IS_ required in order to build properly. See 9074d7ab535caa3abc33bfa14d1e286d358994ed